### PR TITLE
adapt to removed gmail-oauth2

### PR DESCRIPTION
**Pull request by r10s**
i also added a small para for reasoning, trying to be mostly neutral :)

targets https://github.com/deltachat/deltachat-android/issues/3233

### CODE REVIEW

File: _providers/gmail.md [hunk 1]
```diff
@@ -18,16 +18,18 @@ server:
 opt:
   delete_to_trash: true
 before_login_hint: |
-  For Gmail accounts, you need to create an app-password if you have "2-Step Verification" enabled. If this setting is not available, you need to enable "less secure apps".
-last_checked: 2021-08
+  For Gmail accounts, you need to create an app-password and have "2-Step Verification" enabled.
```
Comment (iequidoo) [inline _providers/gmail.md:21]: Better to flip this. Afair it's impossible to create an app password w/o having 2-Step Verification already enabled

**Review by iequidoo — APPROVED** [review]

**Comment by iequidoo** [comment]
Also this link may be helpful here as well, it was hard to find even for me: https://myaccount.google.com/apppasswords

File: _providers/gmail.md (hunk 1 above)
Comment (r10s) [reply _providers/gmail.md]: makes sense, i added a commit

**Comment by r10s** [comment]
> Also this link may be helpful here as well, it was hard to find even for me: https://myaccount.google.com/apppasswords

i added a commit

**Pull request merged**